### PR TITLE
feature: Update code patterns view CY-4755

### PR DESCRIPTION
**Pull request by prcr**
Updates the screenshots that display code patterns with the new code patterns view.

Preview of the updated pages:

https://github.com/codacy/docs/blob/feature/update-code-patterns-CY-4755/docs/repositories-configure/code-patterns.md

https://github.com/codacy/docs/blob/feature/update-code-patterns-CY-4755/docs/repositories/security-monitor.md

### :construction: To do

- [x] Update screenshots with final version
- [x] Wait until the release of the feature

### CODE REVIEW

File: docs/repositories-configure/code-patterns.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ However, you can create a coding standard adapted to your scenario by configurin
 
 1.  Enable or disable the patterns using the checkbox for each pattern.
 
-    To see a detailed explanation about the issues that a pattern can detect and how to fix them, click **Details** on the top right-hand corner of the pattern. Some patterns also allow you to configure the rules:
+    To see an explanation of the issues that a pattern detects and how to fix them, click **Show details**. Some patterns also allow you to configure the rules for detecting issues:
```
Comment (inesgomas) [inline docs/repositories-configure/code-patterns.md:17]: It is weird that you are highlighting the action "show details" but in the screenshot you highlight "hide details"

**Review by inesgomas — APPROVED** [review]

File: docs/repositories-configure/code-patterns.md (hunk 1 above)
Comment (inesgomas) [reply docs/repositories-configure/code-patterns.md:17]: just a detail, I will approve anyway

Comment (prcr) [reply docs/repositories-configure/code-patterns.md:17]: Yes, before the label didn't change when the details were expanded:

image

The main purpose of this screenshot is actually to show the expanded view of the code pattern. So what I'll do is remove the highlight from the "Hide details", as it isn't that important anyways. :ok_hand:

Comment (inesgomas) [reply docs/repositories-configure/code-patterns.md:17]: Coool

**Review by inesgomas — APPROVED** [review]

**Pull request merged**